js/adapt-audio.js

```diff
@@ -72,9 +72,19 @@ define([
     onPageReady: function(view) {
       if (this.audioEnabled) {
           new AudioToggleView({model:view.model});
+          this.showPrompt();
       }
     },
 
+    showPrompt: function() {
+      var pushObject = {
```

Comment:
I'd add an if statement to check for Adapt.course.get('_audio')._prompt. Otherwise it will break the page if the JSON has not been added.